db/seeds.rb

```diff
@@ -13,11 +13,17 @@
   short_symbol: '£', country_code: 'GB')
 Currency.find_or_create_by(id: 4,name: 'Nicaraguan Cordoba', code: 'NIO', symbol: 'NIC$',
   short_symbol: 'C$', country_code: 'NI')
+Currency.find_or_create_by(id: 5,name: 'Mexican Peso', code: 'MXN', symbol: 'MX$',
```

Comment:
For all of these currencies, could you add a space between `id: #,` and `name: 'Currency'`?

Reply:
oops
thanks